### PR TITLE
bpo-28681: Clarify multiple function names in the tutorial

### PR DESCRIPTION
**Pull request by nanjekyejoannah**
Merged ideas from @bitdancer and @terryjreedy in the bpo thread.

https://bugs.python.org/issue28681

### CODE REVIEW

File: Doc/tutorial/controlflow.rst [hunk 1]
```diff
@@ -300,11 +300,10 @@ passed using *call by value* (where the *value* is always an object *reference*,
...
 
-A function definition introduces the function name in the current symbol table.
-The value of the function name has a type that is recognized by the interpreter
-as a user-defined function.  This value can be assigned to another name which
-can then also be used as a function.  This serves as a general renaming
-mechanism::
+A function definition associates the function name with the function object in
+the current symbol table.  The interpreter recognizes the object pointed to by
+that name as a user-defined function.  Other names can also point to that same
+function object and can be used to access the function::
```
Comment (terryjreedy) [inline Doc/tutorial/controlflow.rst]:
```suggestion
function object and can also be used to access the function::
```

**Review by terryjreedy — CHANGES_REQUESTED** [review]
Approved with the addition of 'also' to the part I wrote.  Re-reading, I find it a bit jarring without the addition.  Thank you for attending to this.

**Comment by terryjreedy** [comment]
Guido strongly prefers and the devguide now clearly states that the first message line of a merged commit should be an active tense statement, like the first line of docstrings.  (I would say "Should start with an active verb" except that there might be a short introductory phrase.) This is easier to ensure if the default, the PR title, is written that way.  Hence my edit.

**Review by terryjreedy — APPROVED** [review]

**Comment by nanjekyejoannah** [comment]
Thanks for the review @terryjreedy

**Pull request merged**